python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
If you're parsing the strings passed from the Proxy you can't know without inspecting the object and comparing to the proxy what is needed. (Or at least this is the example of you can't know in advance of parsing the code whether to get the object from in-front of the proxy or behind).